suite-native/module-home/src/components/BiometricsBottomSheet.tsx

```diff
@@ -57,9 +57,12 @@ export const BiometricsBottomSheet = () => {
     const [isVisible, setIsVisible] = useState(false);
     const [biometricsTypes, setBiometricsTypes] = useState<AuthenticationType[]>([]);
 
-    const isFace = biometricsTypes.filter(biometricsType =>
-        [AuthenticationType.FACIAL_RECOGNITION, AuthenticationType.IRIS].includes(biometricsType),
-    );
+    const isFace =
```

Comment:
Can you make this simpler? Seems uneccasary complicated, use pipe operator and some function from here https://mobily.github.io/ts-belt/api/array

Reply:
`biometricsTypes.some(biometricsType => [AuthenticationType.FACIAL_RECOGNITION, AuthenticationType.IRIS].includes(biometricsType));`

Reply:
f9d6552